content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/embeddable/youtube/_cq_design_dialog/.content.xml

```diff
@@ -0,0 +1,192 @@
+            sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+            checked="false"
+            fieldDescription="When checked, enables the ability of content authors to configure this parameter. Otherwise the YouTube default is taken."
+            name="./youtubePlaysInlineEnabled"
+            text="Allow configuration of Inline Playback (iOS)"
+            uncheckedValue="false"
+            value="true">
+            <granite:data
+                jcr:primaryType="nt:unstructured"
+                cmp-embed-dialog-edit-showhidetarget="\[data-cmp-embed-dialog-edit-inline\]"/>
+        </inline>
+        <inlineGroup
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="granite/ui/components/coral/foundation/well">
+            <items jcr:primaryType="nt:unstructured">
+                <enable
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                    checked="false"
+                    fieldDescription="This parameter controls whether videos play inline (on) or fullscreen (off) in an HTML5 player on iOS."
+                    name="./youtubePlaysInlineDefaultValue"
+                    text="Default Value of Inline Playback (iOS)"
+                    uncheckedValue="false"
+                    value="true"/>
+            </items>
+            <granite:data
+                jcr:primaryType="nt:unstructured"
+                cmp-embed-dialog-edit-inline="true"
+                cmp-embed-dialog-edit-showhidetargetvalue="true"/>
+        </inlineGroup>
+        <rel
+            granite:class="toggle-checkbox"
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+            checked="false"
+            fieldDescription="When checked, enables the ability of content authors to configure this parameter. Otherwise the YouTube default is taken."
+            name="./youtubeRelatedVideosEnabled"
+            text="Allow configuration of related videos"
+            uncheckedValue="false"
+            value="true">
+            <granite:data
+                jcr:primaryType="nt:unstructured"
+                cmp-embed-dialog-edit-showhidetarget="\[data-cmp-embed-dialog-edit-rel\]"/>
+        </rel>
+        <relGroup
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="granite/ui/components/coral/foundation/well">
+            <items jcr:primaryType="nt:unstructured">
+                <enable
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                    checked="false"
+                    fieldDescription="If enabled related videos will come from the same channel as the video that was just played, otherwise from any channel."
```

Comment:
It actually works the other way around. Its the only new parameter where the default value is 1 and not 0, see [Youtube parameters](https://developers.google.com/youtube/player_parameters#Parameters). So guess its easier to change the description than add special case in code.

Reply:
Fixed in https://github.com/adobe/aem-core-wcm-components/pull/1301/commits/86df8b09bbad467797d30dbc05f87dc90d360de9.